src/Resources/config/serializer/Symfony.Component.Debug.Exception.FlattenException.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="Symfony\Component\Messenger\Stamp\RedeliveryStamp" exclusion-policy="ALL">
+        <property name="message" type="string" expose="true" />
+        <property name="code" type="int" expose="true" />
+        <property name="previous" type="Throwable" expose="true" />
+        <property name="trace" type="array" expose="true" />
```

Comment:
Trace can contain objects, if there is no mapping file, I think it will fail, maybe exclude it?

Reply:
Alright, will do.